aptos-move/aptos-gas-schedule/src/gas_schedule/aptos_framework.rs

```diff
@@ -269,6 +270,7 @@ crate::gas_schedule::macros::define_gas_parameters!(
 
         [transaction_context_get_txn_hash_base: InternalGas, { 10.. => "transaction_context.get_txn_hash.base" }, 735],
         [transaction_context_get_script_hash_base: InternalGas, "transaction_context.get_script_hash.base", 735],
+        [transaction_context_get_txn_app_hash_base: InternalGas, "transaction_context.get_txn_app_hash.base", 735],
```

Comment:
Do we need this anymore? Since native function would no longer be needed I think.